README.md

```diff
@@ -148,7 +148,7 @@ to all Giant Swarm installations.
     - [v15.0.0-beta2](https://github.com/giantswarm/releases/tree/master/azure/archived/v15.0.0-beta2)
     - [v15.0.0-beta1](https://github.com/giantswarm/releases/tree/master/azure/archived/v15.0.0-beta1)
   - v15.1
-    - [v15.1.0](https://github.com/giantswarm/releases/tree/master/azure/v15.1.0)
+    - [v15.1.0](https://github.com/giantswarm/releases/tree/master/azure/archived/v15.1.0)
     - [v15.1.1](https://github.com/giantswarm/releases/tree/master/azure/v15.1.1)
```

Comment:
Shouldn't 15.1.1 be above 15.1.0?

Reply:
Yes it should, please swap the two lines

Reply:
i guess somebody did that with release, can change it

Reply:
fixed